check_tags.sh

```diff
@@ -15,52 +15,113 @@ if [ "$GITHUB_TOKEN" != "" ]; then
-      HAVE_64_TAR=$(echo $VSCODIUM_ASSETS | jq --arg suffix "x64-$LATEST_MS_TAG.tar.gz" 'map(.name) | contains([$suffix])')
-      if [[ "$HAVE_64_RPM" != "true" ]]; then
-        echo "Building on Linux x64 because we have no RPM"
-        export SHOULD_BUILD="yes"
-      fi
-      if [[ "$HAVE_64_DEB" != "true" ]]; then
-        echo "Building on Linux x64 because we have no DEB"
-        export SHOULD_BUILD="yes"
-      fi
-      if [[ "$HAVE_64_TAR" != "true" ]]; then
-        echo "Building on Linux x64 because we have no TAR"
-        export SHOULD_BUILD="yes"
+    elif [[ "$OS_NAME" == "windows" ]]; then
+      if [[ $VSCODE_ARCH == "arm64" ]]; then
+        HAVE_ARM64_SYS=$(echo $VSCODIUM_ASSETS | jq --arg suffix "$VSCODE_ARCH-$LATEST_MS_TAG.exe" 'map(.name) | contains([$suffix])')
+        HAVE_ARM64_USR=$(echo $VSCODIUM_ASSETS | jq --arg suffix "UserSetup-$VSCODE_ARCH-$LATEST_MS_TAG.exe" 'map(.name) | contains([$suffix])')
+        HAVE_ARM64_ZIP=$(echo $VSCODIUM_ASSETS | jq --arg suffix "win32-$VSCODE_ARCH-$LATEST_MS_TAG.zip" 'map(.name) | contains([$suffix])')
+        if [[ "$HAVE_ARM64_SYS" != "true" ]]; then
+          echo "Building on Windows arm64 because we have no system setup"
+          export SHOULD_BUILD="yes"
+        fi
+        if [[ "$HAVE_ARM64_USR" != "true" ]]; then
+          echo "Building on Windows arm64 because we have no user setup"
+          export SHOULD_BUILD="yes"
+        fi
+        if [[ "$HAVE_ARM64_ZIP" != "true" ]]; then
+          echo "Building on Windows arm64 because we have no zip"
+          export SHOULD_BUILD="yes"
+        fi
+        if [[ "$SHOULD_BUILD" != "yes" ]]; then
+          echo "Already have all the Windows arm64 builds"
+        fi
+      elif [[ $VSCODE_ARCH == "ia32" ]]; then
+        HAVE_IA32_SYS=$(echo $VSCODIUM_ASSETS | jq --arg suffix "$VSCODE_ARCH-$LATEST_MS_TAG.exe" 'map(.name) | contains([$suffix])')
+        HAVE_IA32_USR=$(echo $VSCODIUM_ASSETS | jq --arg suffix "UserSetup-$VSCODE_ARCH-$LATEST_MS_TAG.exe" 'map(.name) | contains([$suffix])')
+        HAVE_IA32_ZIP=$(echo $VSCODIUM_ASSETS | jq --arg suffix "win32-$VSCODE_ARCH-$LATEST_MS_TAG.zip" 'map(.name) | contains([$suffix])')
+        if [[ "$HAVE_IA32_SYS" != "true" ]]; then
+          echo "Building on Windows ia32 because we have no system setup"
+          export SHOULD_BUILD="yes"
+        fi
+        if [[ "$HAVE_IA32_USR" != "true" ]]; then
+          echo "Building on Windows ia32 because we have no user setup"
+          export SHOULD_BUILD="yes"
+        fi
+        if [[ "$HAVE_IA32_ZIP" != "true" ]]; then
+          echo "Building on Windows ia32 because we have no zip"
+          export SHOULD_BUILD="yes"
+        fi
+        if [[ "$SHOULD_BUILD" != "yes" ]]; then
+          echo "Already have all the Windows ia32 builds"
+        fi
+      else # Windows x64
+        HAVE_X64_SYS=$(echo $VSCODIUM_ASSETS | jq --arg suffix "$VSCODE_ARCH-$LATEST_MS_TAG.exe" 'map(.name) | contains([$suffix])')
```

Comment:
It can either match `VSCodiumSetup-x64-$VERSION.exe` or `VSCodiumUserSetup-x64-$VERSION.exe`